website/docs/en/blog/announcing-0-5.mdx

```diff
@@ -183,7 +183,7 @@ Apply entry lazily is deprecating by rspackFuture: [experiments.rspackFuture.dis
 
 When `experiments.rspackFuture.disableApplyEntryLazily` is `false`, `options.entry` can still make valid changes after `rspack(options)` is called, but with `true` it can't, and it's behave the same as webpack5.
 
-This configuration has no effect on users developing applications in Rspack most of the time, but should be noted by developers of Rspack plugins or upper-level frameworks.
+This configuration has no effect on users developing applications in Rspack most of the time, but should be noted by developers of Rspack plugins or higher-level frameworks.
```

Comment:
`announcing-0-5.mdx` is not the focus of the current PR.

This PR has changes to that file, just because I globally replaced upper-level -> higher-level

Reply:
For the definition of higher-level tools /framework, refer to https://vitejs.dev/guide/philosophy#building-frameworks-on-top-of-vite